builtin/logical/pki/backend_test.go

```diff
@@ -4290,7 +4291,7 @@ func TestBackend_RevokePlusTidy_Intermediate(t *testing.T) {
 			"secrets.pki.tidy.cert_store_current_entry":                         0,
 			"secrets.pki.tidy.cert_store_total_entries":                         1,
 			"secrets.pki.tidy.revoked_cert_current_entry":                       0,
-			"secrets.pki.tidy.revoked_cert_total_entries":                       1,
```

Comment:
@cipherboy since they are 2 tidy calls, the first tidy call cleans up the revoked cert and so the metrics will show `revoked_cert_total_entries = 0` in the end, if I add a check between the tidy calls on line 4191 and 4220 both of the following are true: 
```
"secrets.pki.tidy.revoked_cert_total_entries":           1,
"secrets.pki.tidy.revoked_cert_total_entries_remaining": 1,
```

Reply:
I will add this to the test before I rebase